contrib/win32/win32compat/misc.c

```diff
@@ -931,7 +931,7 @@ resolved_path_utf16(const char *input_path)
 		if (changed_req > 0) {
 			wchar_t * resolved_path_new = realloc(resolved_path, 
 				(resolved_len + changed_req + 1) * sizeof(wchar_t));
-			if (resolved_path == NULL) {
+			if (resolved_path_new == NULL) {
```

Comment:
good catch :)

Reply:
:-) Code analysis actually caught this.  It noted the fact that resolved_path_new was never checked for a null value prior to use.